taskqueuebehaviour.lua

```diff
@@ -628,13 +597,13 @@ function TaskQueueBehaviour:BestFactoryPostPositionFilter(factoryName,p , mtype,
 	end
 	if mtype == 'bot' then
 		local vehNetwork = ai.factoryBuilded['veh'][ai.maphandler:MobilityNetworkHere('veh',p)]
-		if vehNetwork and vehNetwork > 0 and vehNetwork < 4 then
+		if vehNetwork and vehNetwork > 0 and vehNetwork < 4 and ai.factoryBuilded['air'][1] > 0 then
```

Comment:
i understand this to mean "build unnecessary bot factories until an air factory is built". if i understand correctly, what's the reason for this?